src/pages/_app.tsx

```diff
@@ -1,8 +1,9 @@
 import type { AppProps } from "next/app";
 import "../../public/global.css";
```

Comment:
why don't we move this to styles folder ?

Reply:
That's a good idea. I think we can do that in the same PR where we migrate from styled-jsx to CSS modules